src/models/quote.js

```diff
@@ -10,6 +11,32 @@ const ExternalError = require('../errors/external-error')
 const balanceCache = require('../services/balance-cache.js')
 const backend = require('../services/backend')
 
+// TODO: Validate amounts are not greater than ledger precision
+function * getPrecisionAndScale (ledger) {
+  log.debug('getPrecisionAndScale', ledger)
+  function throwErr () {
+    throw new ExternalError('Unable to determine ledger precision')
+  }
+
+  let res
+  try {
+    res = yield request(ledger, {json: true})
```

Comment:
Under what circumstances will `request` throw? I thought it would just return an error code if something goes wrong.